mix.exs

```diff
@@ -20,34 +19,27 @@ defmodule Hammer.Mixfile do
   end
 
   def application do
-    # Specify extra applications you'll use from Erlang/Elixir
-    [mod: {Hammer.Application, []}, extra_applications: [:logger, :runtime_tools]]
+    [extra_applications: [:logger]]
   end
 
   defp deps do
     [
       {:credo, "~> 1.7", only: [:dev, :test]},
       {:ex_doc, "~> 0.30", only: :dev},
-      {:dialyxir, "~> 1.3", only: [:dev, :test], runtime: false},
-      {:poolboy, "~> 1.5"}
```

Comment:
are you gonna let each backend handle their own pool?

Reply:
Yes. And ETS doesn't have a pool.